### ISSUE TITLE
Error in the end of the migrations up command

### ISSUE THREAD

**Issue by yaring** [db-migrate/node-db-migrate#331]
When I'm running `NODE_ENV=development db-migrate up`
I'm getting this error
```
[ERROR] TypeError: undefined is not a function
    at Object.<anonymous> (/usr/local/lib/node_modules/db-migrate-mongodb/index.js:439:23)
    at Object.Base.extend.close (/usr/local/lib/node_modules/db-migrate-mongodb/index.js:434:12)
    at onComplete (/usr/local/lib/node_modules/db-migrate/api.js:845:19)
    at /usr/local/lib/node_modules/db-migrate/lib/migrator.js:120:11
    at /usr/local/lib/node_modules/db-migrate/lib/migration.js:339:7
    at processImmediate [as _immediateCallback] (timers.js:367:17)
```

db-migrate version: 0.10.0-beta.6
db-migrate-mongodb version: 1.0.1

**Comment by wzrdtales** [comment]
Please also always tell me your Node version, thank you for reporting so far. As this has something todo with closing the connection I need a bit more details:

https://github.com/db-migrate/mongodb/blob/master/index.js#L439


Can you please tell: 
Can you please post a verbose run?
Do your migrations have worked in another environment before, what are you trying to do currently?
If migrations have not worked can you please show me your migration in which it is throwing this error. You should get this information from the verbose run.

Thank you

**Comment by yaring** [comment]
Node version: v0.12.7
npm version: 2.12.1
Mongodb version: 3.0.0

What I'm trying to do is to migrate manual migrations files to db-migrate plugin, they have worked before when running manually one by one.

db-migrate runs the migrations properly, but at the end, the db conncetion isn't close properly as you see.

Running with verbose:
```
[INFO] Using development settings: { driver: 'mongodb',
  host: 'localhost',
  database: 'building_blocks_development' }
[INFO] require: db-migrate-mongodb
[INFO] connecting
[INFO] connected
[SQL] createCollection [ 'migrations' ]
[INFO] migration table created
[INFO] loading migrations from dir /Users/yarin.goldman/Work/building_blocks_service/migrations
[INFO] loading migrations from database
[SQL] find [ 'migrations' ]
[INFO] No migrations to run
[ERROR] TypeError: undefined is not a function
    at Object.<anonymous> (/usr/local/lib/node_modules/db-migrate-mongodb/index.js:439:23)
    at Object.Base.extend.close (/usr/local/lib/node_modules/db-migrate-mongodb/index.js:434:12)
    at onComplete (/usr/local/lib/node_modules/db-migrate/api.js:845:19)
    at /usr/local/lib/node_modules/db-migrate/lib/migrator.js:120:11
    at /usr/local/lib/node_modules/db-migrate/lib/migration.js:339:7
    at processImmediate [as _immediateCallback] (timers.js:367:17)
```

**Comment by wzrdtales** [comment]
Ok actually the problem is that the connection is already closed at that point due to the current functionality of the mongo db driver, which is planned to be reworked in some future. I'm going to push out a patch to fix this problem as soon as I arrive back home this evening.

Thank you for reporting!

**Comment by yaring** [comment]
@wzrdtales , that will be great!
I'm waiting for this patch :)

**Comment by wzrdtales** [comment]
@yaring You can now update the mongo db driver, patch released under v1.0.2.

**Comment by yaring** [comment]
Awesome! 10x @wzrdtales
